back/app/mailers/application_mailer.rb

```diff
@@ -92,7 +92,8 @@ def default_config
       subject: subject,
       from: from_email,
       to: to_email,
-      reply_to: reply_to_email
+      reply_to: reply_to_email,
+      domain: domain
```

Comment:
I'm not entirely convinced this will work, and I'm especially concerned that it might overwrite the value configured at initialization with `nil` and thus break mailgun mailing when no custom `from_email` (and therefore custom `domain`) is set.

It looks like this might well be the simplest approach, however, based on what I understand from [this thread](https://github.com/mailgun/mailgun-ruby/issues/161#issuecomment-762202877).

I think it would be good to merge this to master and test on our `demo.stg.citizenlab.co` platform, which has mailgun set up. I think this is safe to do, as it's highly unlikely anyone will spot the new setting in AdminHQ and then set it for a customer.

Reply:
> I'm especially concerned that it might overwrite the value configured at initialization with nil and thus break mailgun mailing when no custom from_email (and therefore custom domain) is set

I think that's probably the case with the current code, indeed, adding a comment at the relevant bit.